src/cdk/portal/portal-directives.ts

```diff
@@ -49,6 +49,7 @@ export class TemplatePortalDirective extends TemplatePortal<any> {
  */
 @Directive({
   selector: '[cdkPortalHost], [portalHost]',
+  exportAs: 'cdkPortalHost, portalHost',
```

Comment:
I believe the intention is to remove `[portalHost]` eventually right?  If that is the case, should we not export as `portalHost`?

Reply:
Makes sense. Updated it